src/main/resources/avro/bdg.avdl

```diff
@@ -695,6 +626,17 @@ record Genotype {
    is phased. Should only be populated if phased == true; else should be null.
    */
   union { null, int } phaseQuality = null;
+
+  /**
+   True or false implies that filters were applied and this genotype PASSed or not.
+   While 'null' implies not filters were applied.
```

Comment:
Should we note in the comment here that this value and the next are derived from the FT field of FORMAT?